### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by Nionom** [Rem0o/FanControl.Releases#427] [General]
I just wanted to post something here because this software is amazing. I've been daily-driving it for over half a year now, and I've had zero issues whatsoever. The updates are very frequent, and the software is very lightweight and works really well!

I LOVE the new auto fan curve feature too!

**Comment by emilnym** [comment]
Same here! This software saved my mini-itx build from being a horrendous and noisy piece of steel to a wonderful machine!

**Reply by Gozzima** [reply]
I just installed this software and I found it very easy to setup and it works perfectly.
It allows me to control my case fans based on the GPU temp which is exactly what I was not able to do with Asus Fan Xpert or Corsair ICue and my Nvidia FE GPU. I will also explore mix curves to combine CPU and GPU temp sources.
Thanks a lot to developers !

**Comment by acme64** [comment]
been around since v39, its been absolutely amazing
